lib/browser.ts

```diff
@@ -1074,9 +1071,20 @@ export class ProtractorBrowser extends Webdriver {
       execute_: function(execFn_: Function) {
         this.execPromiseResult_ = this.execPromiseError_ = undefined;
 
-        this.execPromise_ = this.execPromise_.then(execFn_).then(
-            (result: Object) => { this.execPromiseResult_ = result; },
-            (err: Error) => { this.execPromiseError_ = err; });
+        (<any>global).tempEl = (<any>global).browser.rootEl;
```

Comment:
It makes me nervous to add this here. I think this should be a change in the general logic for how we look for the root Element, not something specific to the debugger (which is not just used for elementExplorer, but also other REPLs if I'm remembering correctly.